Dependency DLL's are no longer copied into build output folder

HellUnit opened the pull request.

Commit:
Dependency DLL's are no longer copied into build output folder

lukespragg commented:
This shouldn't make a difference. The deploy.sh script copies only the .exe and the .opj files to a temp Dir, and then archives from that folder that should only contain those two.

Commit:
Fixes #39 Fixes #40
'manifest' variable no longer initialized
NewProjectForm directorytextbox now defaults to 'My Documents'
Fixed NullPointer on trying to load non-existent 'resources.dll'

HellUnit commented:
It probably won't which is why I didn't tag it as a fix. Its just cleaner during manual builds to not clutter up the output directory with unneeded DLL's.
